src/i18n/ru/ui.ts

```diff
@@ -17,4 +17,55 @@ export default UIDictionary({
 	'articleNav.prevPage': 'Назад',
 	'leftSidebar.learnTab': 'Изучить',
 	'leftSidebar.referenceTab': 'Ссылки',
+	'404.content': 'Эта страница не в нашей солнечной системе.',
+	'404.linkText': 'Доставь меня домой.',
+	'404.title': 'Не найдено',
+	'a11y.sectionLink': 'Раздел, озаглавленный',
+	'a11y.skipLink': 'Перейти к контенту',
+	'checklist.or': 'или',
+	'cms.navTitle': 'Больше гайдов по CMS',
+	'contributors.seeAll': 'Посмотреть всех контрибьюторов',
+	'deploy.altSectionTitle': 'Больше гайдов по деплою',
+	'deploy.filterLabel': 'Отфильтровать по типу деплоя',
+	'deploy.sectionTitle': 'Гайды по деплою',
+	'deploy.ssrTag': 'SSR',
+	'deploy.staticTag': 'Static',
+	'fallbackContent.linkText': 'Узнать больше о том, как вы можете помочь',
+	'fallbackContent.notice':
+		'Эта страница ещё не доступна на вашем языке, поэтому вы видите английскую версию. Вы можете помочь перевести её!',
```

Comment:
This could probably say “not yet available in Russian” instead of “your language”. We used “your language” in the English version so it could apply in all languages where we hadn’t translated this yet, but for the translation the specific version probably makes sense.